pkg/config/log_format_serverless.go

```diff
@@ -0,0 +1,21 @@
+// +build serverless
+
+package config
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/cihub/seelog"
+)
+
+// buildCommonFormat returns the log common format seelog string
+func buildCommonFormat(loggerName LoggerName) string {
+	return fmt.Sprintf("%%Date(%s) | %s | %%LEVEL | %%Msg%%n", getLogDateFormat(), loggerName)
+}
```

Comment:
I'm just curious, why do we remove contextual information from agent log when running in the serverless context ?
I guess here it's not a problem not to check for JMXFetch.

Reply:
We are mimicking the logs that users are used to have while running functions in AWS Lambda (through our clients libraries for instance).